package.json

```diff
@@ -106,6 +107,7 @@
     "intl-relativeformat": "^6.4.3",
     "is-nan": "^1.3.0",
     "js-yaml": "^3.13.1",
+    "kind-of": "^6.0.3",
```

Comment:
It is not used from dependency dependencies.  You need to use the resolutions property.